### DISCUSSION TITLE
Realtek Audio Console does not work/launch after restart

### DISCUSSION THREAD

**Discussion by ghost** [pal1000/Realtek-UAD-generic#138] [General]
After the driver is installed, the Audio Console works, but then when I restarted it no longer works. Any idea what happened?

Windows 11. Downloaded and used 6.0.9481.1 with RealtekAudioControl_1.41.290

**Comment by pal1000** [comment]
Maybe Windows update pulls in a driver behind your back.

**Reply by ghost** [reply]
Driver updates are disabled in "Device Installation Settings"

So I did a little digging and it turns out you need to also change some group policies and regedit keys in order for windows to install auto install driver updates. Very sneaky indeed.

https://www.tenforums.com/drivers-hardware/133482-device-manager-update-driver-issue-2.html
https://www.minitool.com/news/disable-automatic-driver-updates-win-10-009.html
https://forums.guru3d.com/threads/windows-10-registry-tweak-for-disabling-drivers-auto-update-controversy.418033/

I guess it is fixed for now. Thanks for the suggestion!
